### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Shailseen**
image
image

### CODE REVIEW

File: pages/homepage/homepage.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            <div class="badge badge-icon-card">6</div>
+          </div>
+        </div>
+      </nav>
+        <section class="menu-container">
+          <div class="text-overlay-component">
+            <img
+              class="img-responsive menu-image"
+              src="/assets/preview.jpg"
+              alt=""
```
Comment (prakashsakari) [inline pages/homepage/homepage.html]: you can a value in the alt attribute

**Review by prakashsakari — APPROVED** [review]
Added a comment, rest looks good to me

File: pages/homepage/homepage.html (hunk 1 above)
Comment (Shailseen) [reply pages/homepage/homepage.html]: Okay , i will add it.

File: pages/homepage/homepage.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// var slideIndex = 0;
```
Comment (ishitakeshawani) [inline pages/homepage/homepage.js]: why did everything in comment?

**Review by ishitakeshawani — APPROVED** [review]

File: pages/homepage/homepage.js (hunk 2 above)
Comment (Shailseen) [reply pages/homepage/homepage.js]: I forget to remove these.

**Pull request merged**